.ci/atime/tests.R

```diff
@@ -118,7 +118,37 @@ test.list <- atime::atime_test_list(
       data.table:::setattr(L, "class", NULL)
       data.table:::setDT(L)
     },
-    Slow = "c4a2085e35689a108d67dacb2f8261e4964d7e12", # Parent of the first commit in the PR that fixes the issue (https://github.com/Rdatatable/data.table/commit/7cc4da4c1c8e568f655ab5167922dcdb75953801)
-    Fast = "1872f473b20fdcddc5c1b35d79fe9229cd9a1d15") # Last commit in the PR that fixes the issue (https://github.com/Rdatatable/data.table/pull/5427/commits)
+    Slow = "c4a2085e35689a108d67dacb2f8261e4964d7e12",  # Parent of the first commit in the PR that fixes the issue (https://github.com/Rdatatable/data.table/commit/7cc4da4c1c8e568f655ab5167922dcdb75953801)
+    Fast = "1872f473b20fdcddc5c1b35d79fe9229cd9a1d15"), # Last commit in the PR that fixes the issue (https://github.com/Rdatatable/data.table/pull/5427/commits)
+  "fread(colClasses=list(Date)) improved in #6107" = atime::atime_test(
+    N = 10^seq(1, 7),
+    setup = {
+      DT = data.table(date=.Date(sample(20000, N, replace=TRUE)))
+      tmp_csv = tempfile()
+      fwrite(DT, tmp_csv)
+    },
+    expr = data.table::fread(tmp_csv, colClasses=list(Date='date')),
+    Slow = "d19bfef7026f25bb2d36c17879187d09bcb2b2c3",
```

Comment:
@MichaelChirico Can you please add a comment where the SHAs came from? (reviewing with Toby)

Reply:
`Slow` I pulled from `master` around when I filed this PR. `Fast` is the first commit of this PR. Is that worth documenting? Is there some better approach I should be aware of?

Reply:
Sounds reasonable enough to me, but I think @tdhock prefers to have comments mentioning that information beside those lines for consistency with how the other tests are documented or for others to know quickly why those commit SHAs are being used

Reply:
to make it easy to double check where the commit IDs come from, it is best if they can be referenced relative to a PR, like the existing comments, "last commit in PR#xyz (URL)" etc

Reply:
Per recent discussion, dropped the atime edit here; I have it ready in branch `fread-date-idate-atime` to submit as a follow-up. That'll resolve the issues with git hash.